### PR TITLE
Resolving #21 issue. Adding ko version and simple release script

### PR DESCRIPTION
**Pull request by cezkuj**
Output from version command implemented as we have discussed in comments, easily we can extend it with other build information from govvv, if needed.
Also, added basic release script, I didn't have any guidelines so I added what seemed valid for me, let's discuss if it should be improved.
Cheers!

### CODE REVIEW

File: hack/release.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Usage:
+#   hack/release.sh $VERSION
+#
+# Example:
+#   hack/release.sh v0.2
+
+VERSION=$1
+
+KO_ROOT="$(cd "$(dirname "$0")" && pwd)/.."
+govvv build -o $KO_ROOT/build/ko $KO_ROOT/cmd/ko -version $VERSION
```
Comment (jonjohnsonjr) [inline hack/release.sh]: Perhaps add a `go get github.com/ahmetb/govvv` before this?

Comment (cezkuj) [reply hack/release.sh]: Good point. Added.

**Comment by cezkuj** [comment]
@jonjohnsonjr @ImJasonH 

I've updated branch so it doesn't have conflicts anymore. Could you review it?

**Comment by ImJasonH** [comment]
I think we want to run `git rev-parse` in `ko`'s directory in `GOPATH`, not in the current working directory, which is probably another repo and not ko's.

**Comment by cezkuj** [comment]
Hmm, agreed but, using GOPATH wouldn't be possible if migrated to go modules. It is definitely drawback - not sure if valid for now.

For the time being we could use git dir based on GOPATH. Smth like that:
```go
gitDir := fmt.Sprintf("--git-dir=%v/src/github.com/google/ko/.git", os.Getenv("GOPATH"))
hash, err := exec.Command("git", gitDir, "rev-parse", "HEAD").Output()
```

**Pull request merged**